src/V3Inline.cpp

```diff
@@ -476,175 +439,225 @@ class InlineRelinkVisitor final : public VNVisitor {
 };
 
 //######################################################################
-// Inline state, as a visitor of each AstNode
+// Module inliner
+
+namespace ModuleInliner {
+
+// Connect the given port 'nodep' (being inlined into 'modp') to the given
+// expression (from the Cell Pin)
+void connectPort(AstNodeModule* modp, AstVar* nodep, AstNodeExpr* pinExprp) {
+    UINFO(6, "Connecting " << pinExprp);
+    UINFO(6, "        to " << nodep);
+
+    // We can do this 2 ways. Either add a continuous assignment between
```

Comment:
```suggestion
    // Can do this 2 ways. Either add a continuous assignment between
```

Prefer comments avoid assumptions reader is the programmer.